db/migration/migration.go

```diff
@@ -0,0 +1,99 @@
+// Copyright (C) 2015 NTT Innovation Institute, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package migration
+
+import (
+	"fmt"
+
+	"database/sql"
+	"github.com/cloudwan/gohan/log"
+	"github.com/cloudwan/gohan/util"
+	"github.com/pressly/goose"
+	"os"
+	"path"
+)
+
+var logger = log.NewLogger()
+
+func LoadConfig(configFile string) (err error) {
+	config := util.GetConfig()
+
+	err = config.ReadConfig(configFile)
+
+	if err != nil {
+		fmt.Printf("error: failed to load config: %s\n", err.Error())
```

Comment:
When running gohan in migration mode, we assume that we operate directly with gohan. All messages are put to stdout andwe do not have to setup logging and provide proper configuration files.